src/EnergyPlus/SingleDuct.cc

```diff
@@ -5310,6 +5315,8 @@ namespace SingleDuct {
         if (Contaminant.GenericContamSimulation) {
             Node(OutletNode).GenContam = Node(InletNode).GenContam;
         }
+        // set OA volume flow rate report variable
+        this->CalcOutdoorAirVolumeFlowRate();
```

Comment:
this new member function is called during update() function as it is used to update a report variable.